doc/development/python-release.md

```diff
@@ -0,0 +1,67 @@
+# Python Release Process
+
+This document describes the general way in which Python packages are released. 
+
+## Prerequistes: 
+
+- Optionally create a new virtual environment for the `bumpversion`  package, and other release tools
+- Install the bumpversion package , `pip install bumpversion`
+- Following git-flow process
+- Running Travis in the repo, i.e. `.travis.yml` exists and has the `deploy` section defined, (see below)
+- bumpversion.cfg exists, example below
```

Comment:
```suggestion
- `.bumpversion.cfg` exists, example below
```

Reply:
Maybe also add that they shouldn't have any `bumpversion` stuff in `setup.cfg`: bumpversion looks in `.bumpversion.cfg` first and `setup.cfg` second but we should just standardize on always using `.bumpversion.cfg`.